lib/upgrade_format_task.rb

```diff
@@ -46,18 +52,16 @@ def upgrade(store, name, options = {})
     if !errors.empty?
       Machinery.logger.error errors.join("\n")
       exception = Machinery::Errors::UpgradeFailed.new(errors.join("\n") +
-        Hint.to_string(:upgrade_format_force, name: name || "--all"))
+                                                           Hint.to_string(:upgrade_format_force, name: name || "--all"))
       raise exception
     end
 
     if options[:all]
       if migrations_done > 0
-        Machinery::Ui.puts "Upgraded #{migrations_done} system descriptions successfully."
-      else
+        Machinery::Ui.puts "Upgraded #{migrations_done} #{"system description".pluralize(migrations_done)}."
```

Comment:
Line is too long. [108/100]<br>Prefer single-quoted strings inside interpolations.